### DISCUSSION TITLE
What's the expired case behaviour?

### DISCUSSION THREAD

**Discussion by http-alba** [globaleaks/globaleaks-whistleblowing-software#3632] [Q&A]
We have a tennant with an admin user concerned about recovering a case that has already expired.
Looking through the documentation I can't find the exact way to retrieve them back or find them, unless the user saves them before the expiration data limit.
Is there something I'm missing here?
Thanks a lot for your help!

**Comment by evilaliv3** [comment]
Hello!

GlobaLeaks was born  as a communication tool between whistleblowers and recipients and the data expires and is deleted according to the data retention policy.
It is up to users to decide if they want to prolong (postpone) the expiration date or if it is fine that the data expires forever. Eventually they can download and archive it offline at any time.
